pkg/registry/core/service/ipallocator/allocator.go

```diff
@@ -254,7 +254,7 @@ func calculateIPOffset(base *big.Int, ip net.IP) int {
 func RangeSize(subnet *net.IPNet) int64 {
```

Comment:
Alternatively, should the function just return the # networks, 0+, and then the caller can check for validity of the number of the range? Right now, it determines the range of hosts AND checks for exceeding a maximum, but not a minimum. Looks like some callers are doing checking.